dallinger/command_line.py

```diff
@@ -836,13 +843,11 @@ def export(app, local):
 
     for table in all_tables:
         subprocess.check_call(
-            "psql -d dallinger --command=\"\\copy " + table + " to \'" +
+            "psql -d " + app_name(id) +
+            " --command=\"\\copy " + table + " to \'" +
             os.path.join(subdata_path, table) + ".csv\' csv header\"",
             shell=True)
```

Comment:
Agreed — there's an open issue on that (#353) and I started a branch `no-shell-subprocess` that gets rid of most of them.